organization/components/UserTable.tsx

```
@@ -0,0 +1,88 @@
+              {canAssignRoles && (
+                <th className="px-3 py-3.5 text-right text-sm font-semibold text-gray-900 pr-6">
+                  Actions
+                </th>
+              )}
+            </tr>
+          </thead>
+          <tbody className="divide-y divide-gray-200">
+            {users.map(user => (
+              <tr key={user._id} className="hover:bg-gray-50 transition-colors">
+                <td className="whitespace-nowrap py-4 pl-6 pr-3 text-sm text-gray-900">
+                  {user.first_name + ' ' + user.last_name}
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)